lib/assets/schemas/tools/umm-t-preview.json

```diff
@@ -6,28 +6,37 @@
       "fields": [
         {
           "key": "Name"
-        }, {
+        },
```

Comment:
just curious, why did you change this file in this way?

Reply:
I think I noticed that the entire form had some indentation issues, so I did  a full reformat, but looks like maybe it did a little more than anticipated.